SR2019/2019-03-11-competition-information.md

```diff
@@ -0,0 +1,53 @@
+---
+to: Student Robotics 2019
+subject: Competition Informatiom
+---
+
+Hi all,
+
+There's just 26 days between now and the competition, and we're getting the final things ready for Student Robotics 2019. We hope your robots are battle-ready, and you're well on your way to entering a robot capable of winning!
+
+We have some final information for you for the competition.
+
+## Reduced mobility
+
+The team pits for this year's competition are spread over 3 floors. If any members of your team have reduced mobility, please let us know as soon as possible so we can account for this when placing teams.
+
+## Media Consent Form and Tickets
+
+As with previous years, we require you to sign a media consent form. These media consent forms allow us to use videos and photographs taken at the event for future marketing purposes.
+
+[Download our Media Consent form]()
+
+If a member of your team is under the age of 16, they will need to get a parent or guardian to sign on their behalf.
+
+Once these forms have been completed, please send them to `teams@studentrobotics.org` with the word 'media-consent' in the subject line, to ensure they won't be missed. We will have media consent forms available at the event, but pre-processing them makes the reception process faster.
+
+Once your form has been processed, you'll receive an automated email response with your ticket. These tickets must be bought with you to the competition (printed or on a phone is fine).
+
+## Rules Update
+
+We recently made our first update to the rules. This contained some important, but minor changes:
+
+- Change the documented hashtag from `#srobo2019` to `#srobo19` to follow what was being used internally
```

Comment:
It seems odd to put this minor change first? The robot-in-zone and supporting equipment changes strike me as the most important.

Reply:
It's actually in chronological order IIRC, but yes let's re-shuffle it.